docs/userguide.md

```diff
@@ -285,6 +291,167 @@ kubefedctl disable <FederatedTypeConfig Name> --delete-from-api
 
 **WARNING: All custom resources for the type will be removed by this command.**
 
+## Federating a target resource
+Apart from `enabling` and `disabling` a `type` for `propagation` as specified in the previous 
+section, `kubefedclt` can also be used to `federate` a target resource of an API type.
+We define the term `federate` [here](concepts.md#federation-v2-concepts) and use the command keyword 
+`federate` in `kudefedctl` with similar meaning.
+
+`kubefedctl federate` creates a federated resource from a kubernetes resource wrapping the 
+target kubernetes resource spec, includes a default placement (which includes all clusters) and 
+includes no overrides. The federated resource is then a resource in the common API surface 
+(federation API) to be used to manage the target resources in the individual clusters. 
+The default placement means the resource will be propagated to all member clusters. If a cluster
+has a pre existing resource of this type with the same name and in the same namespace, it will 
+be adopted by this federated resource. The adoption is a default behaviour and can be overridden 
+in the federation controller manager config using the field `skip-adopting-resources: true`. 
+Please take note that this setting is global for the given federation control plane and would 
+apply to all federated resources.
+
+***Syntax***
+```bash
+kubefedctl federate <target kubernetes API type> <target resource> [flags]
+```
+
+The `<target kubernetes API type>` above can be the Kind (e.g. `Deployment`), plural name
+(e.g. `deployments`), group-qualified plural name (e.g `deployment.apps`), or short name
+(e.g. `deploy`) of the intended target API type.
+The `<target resource>` above is the name of the target resource which must exist in the
+same cluster hosting the federation API. If the flag `--namespace` is not specified, the
+resource will be searched for in the namespace `default`. Please take note that `--namespace`
+flag is of no meaning when federating a `namespace` itself and is discarded even if specified. 
+Please check the next section for more details about [federating a namespace](#federate-a-namespace).
+
+***Example:***
+Federate a resource named "my-cm" in namespace "my-ns" of kubernetes type "configmaps" 
+```bash
+kubefedctl federate cm my-cm -n my-ns
+```
+
+An installed `FederatedTypeConfig` matching the `<target kubernetes API type>` (named as 
+`<target API Resource name>.<target API Resource group>`) is a prerequisite for the `federate
+resource` to work if the federated resource would be created in the federation API surface.
```

Comment:
s/federation/kubefed/

Please ensure federation is replaced with kubefed or Kubefed where appropriate.

Reply:
Updated!